docroot/modules/custom/cs_generated_content/src/CsGeneratedContentHelper.php

```diff
@@ -27,4 +28,39 @@ public static function randomRealWebform() {
     return count($entities) > 0 ? reset($entities) : NULL;
   }
 
+  /**
+   * Get image variation.
+   *
+   * @param \Drupal\file\FileInterface $file
+   *   File entity.
+   * @param int $scale
+   *   Scale constant.
+   *
+   * @return \Drupal\file\FileInterface
+   *   Created managed file.
+   */
+  public static function resizeImagePng(FileInterface $file, $scale) {
```

Comment:
@AlexSkrypnyk I have added this helper function to generate png image variations.